wcivf/apps/elections/tests/test_election_views.py

```diff
@@ -39,3 +40,45 @@ def test_election_detail_view(self):
         self.assertEqual(response.status_code, 200)
         self.assertTemplateUsed(response, "elections/election_view.html")
         self.assertContains(response, self.election.nice_election_name)
+
+class ElectionPostViewTests(TestCase):
+    def setUp(self):
+        self.election = ElectionFactory(
+            name="Adur local election",
+            election_date="2021-05-06",
+            slug="local.adur.churchill.2021-05-06",
+        )
+        self.post = PostFactory()
+        self.post_election = PostElectionFactory(election=self.election, post=self.post)
+        people = [PersonFactory() for p in range(5)]
+        for person in people:
+                PersonPostFactory(
```

Comment:
This looks like it is over-indented - should be 4 spaces. Although `black` will reformat this (and any other similar formatting errors like line length) for you. I added some _brief_ notes about running it [here](https://github.com/DemocracyClub/WhoCanIVoteFor/blob/master/INSTALL.md#code-formatting) but also might be worth getting `pre-commit` set up so that it runs black before each commit.